src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/Properties/CSharpProjectConfigurationProperties.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using VSLangProj110;
+using VSLangProj80;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Properties
+{
+    [Export(ExportContractNames.VsTypes.ConfiguredProjectPropertiesAutomationObject)]
+    [Order(Order.Default)]
+    [AppliesTo(ProjectCapability.CSharp)]
+    public class CSharpProjectConfigurationProperties : AbstractProjectConfigurationProperties,
+        CSharpProjectConfigurationProperties3,
```

Comment:
6 goes all the way still 4 and stops there. This is true for both VB and C#.
C# implements `CSharpProjectConfigurationProperties3` which implements `ProjectConfigurationProperties3`.
VB implements just `ProjectConfigurationProperties3` and there is no `VBProjectConfigurationProperties3`.